### PR TITLE
Rename function for displaying error message

### PR DESCRIPTION
**Pull request by vandenman**
Fixes https://github.com/jasp-stats/jasp-test-release/issues/1449

So I git blamed my way back into the past and found https://github.com/jasp-stats/jasp-desktop/commit/a22a36809e03a6b86998cceaa526a65b5d5d2291, looks like this function has been incorrectly called for 2 years!

The function that should be called is defined here:

https://github.com/jasp-stats/jaspDescriptives/blob/708c35adc3105eec28b3b8e5080656d5165adc81/R/descriptives.R#L911-L921

The new output is not that super great, but at least it doesn't crash:

image

basically, each failing subplot shows an error message. In this case, the error messages are also always the same, but in general, it could also be that only 1 subplot shows an error message while the other ones are actually showing a nice plot.

### CODE REVIEW

**Comment by TimKDJ** [comment]
It will have called https://github.com/jasp-stats/jaspRegression/blob/e7bc8242d39dadd83eb67e1a9dd80221860c25f3/R/correlation.R#L1419 before we switched to modules

**Review by JorisGoosen — APPROVED** [review]
